lib/gruf/lightstep/server_interceptor.rb

```diff
@@ -29,7 +29,7 @@ def call(&_block)
 
         result = nil
 
-        whitelist = options.fetch(:whitelist, []).map(&:to_s).map(&:to_sym)
+        allow_list = options.fetch(:allowlist, []).map(&:to_s).map(&:to_sym)
```

Comment:
As discussed offline, I have a preference for `allowlist` over `allow_list` but definitely not a blocker.